README.md

```diff
@@ -42,65 +44,84 @@ postgresql_pg_hba_conf:
     auth_method: ident
 ```
 ### postgresql_server_conf
-Usage of `postgresql_server_conf` adds defined values at the end of postgresql.conf.
-So the default ones are overwritten.
+The content of the `postgresql_server_conf` variable is added to the end of 
+the `/var/lib/pgsql/data/postgresql.conf` file. As a result, the default 
+settings are overwritten.
 ```yaml
 postgresql_server_conf:
   ssl: on
   shared_buffers: 128 MB
   huge_pages: try
 ```
 ### postgresql_ssl_enable
-To set up ssl connection it's necessary to set up `postgresql_ssl_enable` variable and provide server certificate and key.
+To set up a SSL/TLS connection, set the `postgresql_ssl_enable` variable to 
+`True`  and provide a server certificate and a private key.
```

Comment:
```suggestion
`true`  and provide a server certificate and a private key.
```
to be consistent with what you have in the command block below, and generally in system roles we use `true` and `false` for boolean vars.